### PR TITLE
Bugfix/evancole untitled casting

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR that leverages the PR #23 and builds off [PR #25 for quickbooks_source](https://github.com/fivetran/dbt_quickbooks_source/pull/25)

**What change(s) does this PR introduce?** 

This PR changes the ID casting from the previous release from `type_int` to `type_string`. This is needed as other users have found that some of these ID fields are in fact strings. To future proof this, we will cast all IDs as strings to ensure compatibility for the majority of users.

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [X] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

Since we are making such a large change to the package that will change the datatype of a number of fields that could be used to join on downstream models, this will result in a breaking change.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [X] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [X] Yes, Issue/Feature [#23](https://github.com/fivetran/dbt_quickbooks_source/issues/23)
- [ ] No 

**How did you test the PR changes?** 

 

- [X] CircleCi  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🧶 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Review by fivetran-sheringuyen — APPROVED** [review]
Just minor versioning comments @fivetran-joemarkiewicz but ow lgtm!

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Include in your `packages.yml`
 ```yaml
 packages:
   - package: fivetran/quickbooks
-    version: [">=0.4.0", "<0.5.0"]
+    version: [">=0.5.0", "<0.4.0"]
```
[the remark this answers is not in the document]
Comment (fivetran-joemarkiewicz) [reply README.md:37]: Good catch, this versioning would not work lol updated

File: packages.yml [hunk 2]
```diff
@@ -1,3 +1,6 @@
 packages:
-- package: fivetran/quickbooks_source
-  version: [">=0.4.0", "<0.5.0"]
+# - package: fivetran/quickbooks_source
+#   version: [">=0.4.0", "<0.5.0"]
+  - git: https://github.com/fivetran/dbt_quickbooks_source.git
+    revision: bugfix/evancole-untitled-casting
+    warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:6]:
```suggestion
```

File: packages.yml [hunk 3]
```diff
@@ -1,3 +1,6 @@
 packages:
-- package: fivetran/quickbooks_source
-  version: [">=0.4.0", "<0.5.0"]
+# - package: fivetran/quickbooks_source
+#   version: [">=0.4.0", "<0.5.0"]
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:3]:
```suggestion
 - package: fivetran/quickbooks_source
   version: [">=0.5.0", "<0.6.0"]
```

**Pull request merged**